eZ/Publish/Core/REST/Server/Tests/Output/ValueObjectVisitor/RootTest.php

```diff
@@ -28,7 +29,36 @@ public function testVisit()
 
         $generator->startDocument(null);
 
-        $role = new Root();
+        $resources = array(
```

Comment:
I wouldn't bother testing so many. You can update the test to stop expecting so many items.

The interesting thing to cover would be the various cases that change the visitor's behaviour on the node generation.